pkg/server/apiutil/indexnames.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2024 The Cockroach Authors.
+//
+// Use of this software is governed by the CockroachDB Software License
+// included in the /LICENSE file.
+
+package apiutil
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/pkg/roachpb"
+)
+
+// IndexNames and IndexNamesList are utilities for representing the indexes
+// found in descriptors in human readable formats.
+type IndexNames struct {
+	Database string
+	Table    string
+	Index    string
+	Span     roachpb.Span
+}
+
+func (idx IndexNames) String() string {
+	return fmt.Sprintf("%s.%s.%s", idx.Database, idx.Table, idx.Index)
+}
+
+type IndexNamesList []IndexNames
+
+// ToOutput is a simple function which returns a set of de-duplicated,
+// fully referenced database, table, and index names depending on the,
```

Comment:
nit: extra comma end of line

Reply:
Would it be useful at all to return these values sorted?

Reply:
Right now they will be sorted, but by how they appear in the range. Do you feel that having them sorted by name would look cleaner?